Resources/Prototypes/_Funkystation/BloodCult/cult_abilities.yml

```
@@ -0,0 +1,37 @@
+  id: CultStun
+  healthDrain: 10
+  doAfterLength: 5
+  actionPrototypes:
+  - ActionCultistStun
+
+- type: cultAbility
+  id: CultTwistedConstruction
+  healthDrain: 12
+  doAfterLength: 5
+  actionPrototypes:
+  - ActionCultistTwistedConstruction
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)